src/app/modules/group/pages/group-access/group-access.component.html

```diff
@@ -27,6 +29,53 @@
       </ng-container>
     </ng-container>
   </alg-section>
+
+  <alg-section
+    icon="fa fa-book"
+    i18n-label label="Specific permissions given to the group on content"
+  >
+    <ng-container *ngIf="permissionState$ | async as permissionState">
+      <div class="spinner" *ngIf="permissionState.isFetching">
+        <alg-loading size="medium"></alg-loading>
+      </div>
+
+      <alg-error
+        *ngIf="permissionState.isError"
+        i18n-message message="Error while loading permissions."
+        [showRefreshButton]="true"
+        refreshButtonType="refresh"
+        (refresh)="refreshPermissions()"
+      ></alg-error>
+
+      <ng-container *ngIf="permissionState.isReady">
+        <ng-container *ngIf="permissionState.data.length > 1; else noPermissions">
+          <div class="item" *ngFor="let grantedPermission of permissionState.data">
+          <div>
+            <i class="icon fa fa-folder"></i>
+            <strong>
+              {{ grantedPermission.item.title }}.
+            </strong>
+            <span i18n>Given by</span>
+            <span
+              class="alg-link"
+              (click)="onGroupClick(grantedPermission.sourceGroup.id)"
+              *ngIf="grantedPermission.sourceGroup.id !== group?.id; else currentGroup">
+                {{ grantedPermission.sourceGroup.name }}
+            </span>
+            <ng-template #currentGroup>
+              <span i18n> this group</span>
+            </ng-template>
```

Comment:
The translation will be problematic here because "Given by" and "this group"/{group name} are split into 2 different translations, it breaks the structure of the sentence.
To handle it in a i18n-friendly way, we could (for instance) apply the if higher:
```html
<ng-container *ngIf="is different group condition …; else currentGroup">
  <ng-container i18n>
    <span>Given by</span>
    <span class="alg-link" …>{{ [source group name] }}</span>
  </ng-container>
</ng-container>
<ng-template #currentGroup>
  <span i18n>Given by this group</span>
</ng-template>
```

Reply:
Fixed